### ISSUE TITLE
Playground?

### ISSUE THREAD

**Issue by ericcornelissen** [sh7dm/vue-simple-icons#26]
Do we want to maintain the playground as part of the project or was it just a way of testing the component early on? I personally see no direct benefit of keeping the `./playground` directory around 🤔

**Comment by sh7dm** [comment]
@ericcornelissen I think it will help with visual testing

**Comment by ericcornelissen** [comment]
I agree that it can be useful for that. One problem I'm foreseeing is with the way it is setup now people will just add an icon they want to test and then probably commit, which is not desirable.

Also, I don't really understand everything in the folder, and it feels a bit overkill for a single page with some icons. Care to explain how the folder is intended to work? For example, why is there a `yarn run test` and `yarn run lint` command just for the playground, shouldn't that be included in the commands in the repository root?

**Comment by sh7dm** [comment]
it's just generated using vue-cli

**Comment by ericcornelissen** [comment]
So, what are the `yarn run test` and `yarn run lint` commands for?

Unfortunately I cannot test things myself right now as `yarn install` fails ⬇️ 

```
$ yarn install
yarn install v1.12.3
error An unexpected error occurred: "Invalid value type 630:18 in D:\\Workspace\\vue-simpleicons\\playground\\yarn.lock".
info If you think this is a bug, please open a bug report with the information provided in "D:\\Workspace\\vue-simpleicons\\playground\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```
